pkgs/by-name/cl/cloak-pt/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule {
+  pname = "Cloak";
+  version = "2.8.0";
+
+  src = fetchFromGitHub {
+    owner = "BANanaD3V"; # Actual upstream (cbeuw) got a go.mod issue so we use this
+    repo = "Cloak";
+    rev = "6b08af0";
+    hash = "sha256-2nk/sz2pNVt7rM+6omj+FK1U039swFqjHmnE4n3YC4E=";
+  };
+
+  vendorHash = "sha256-xyzFsk1frnj+HbYpXTjrZzD1NM5mokNv6maCA4CsT/w=";
+
+  doCheck = false;
+
+  ldflags = [
+    "-X main.version=master(6b08af0)"
+  ];
+  meta = with lib; {
+    homepage = "https://github.com/cbeuw/Cloak/";
+    description = "Cloak is a pluggable transport that enhances traditional proxy tools like OpenVPN to evade sophisticated censorship and data discrimination.";
```

Comment:
The description should not start with the package name

See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes